### PR TITLE
Cloud development CLI tool

### PR DESCRIPTION
**Pull request by nataliemona**
Initial cloud infrastructure setup.
More details including description and setup in `cdk/README.md`

Sorry this PR got quite large. The best way to review it, rather than looking at the code, would be to try to follow the README instructions and run it.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,5 +1,16 @@
 [flake8]
-ignore = E203, E266, E501, W503, F403, F401, C901
-max-line-length = 79
-max-complexity = 18
-select = B,C,E,F,W,T4,B9
\ No newline at end of file
+ignore =
```
Comment (nataliemona) [inline .flake8:2]: Don't ignore F401 unused imports warning.

File: backend/core/search.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from elasticsearch_dsl.connections import connections
-from elasticsearch_dsl import Document, Text, Date
+from elasticsearch_dsl import Document, Text
```
Comment (nataliemona) [inline backend/core/search.py:2]: Removed unused code.

File: backend/ctgov/api/serializers.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
-import json
 from rest_framework import serializers
-from ctgov.models import BriefSummaries, BasicSearchM, Studies, Facilities
+from ctgov.models import BriefSummaries, BasicSearchM, Facilities
```
Comment (nataliemona) [inline backend/ctgov/api/serializers.py:2]: Removed unused code.

File: backend/ctgov/api/urls.py [hunk 4]
```diff
@@ -1,5 +1,4 @@
-from django.conf.urls import url
-from django.urls import path, include
```
Comment (nataliemona) [inline backend/ctgov/api/urls.py:2]: Removed unused code.

File: backend/ctgov/api/views.py [hunk 5]
```diff
@@ -1,16 +1,13 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import status
-from rest_framework import permissions
-from ctgov.models import BriefSummaries, BasicSearchM, Studies, Facilities
```
Comment (nataliemona) [inline backend/ctgov/api/views.py:5]: Removed unused code.

File: backend/ctgov/documents.py [hunk 6]
```diff
@@ -2,7 +2,7 @@
 from django_elasticsearch_dsl.registries import registry
 from elasticsearch_dsl import analyzer, tokenizer
 from ctgov.models import Documents
-from .models import Sponsors, ReportedEvents, Studies, BasicSearch
```
Comment (nataliemona) [inline backend/ctgov/documents.py:5]: Removed unused code.

File: backend/ctgov/urls.py [hunk 7]
```diff
@@ -1,4 +1,3 @@
-from django.conf.urls import url
```
Comment (nataliemona) [inline backend/ctgov/urls.py:1]: Removed unused code.

**Review by swt144 — APPROVED** [review]

**Pull request merged**